app/pulp/app/response.py

```diff
@@ -0,0 +1,32 @@
+from gettext import gettext as _
+
+from rest_framework.response import Response
+
+
+class OperationPostponedResponse(Response):
+    """
+    An HTTP response class for returning 202 and a list of spawned tasks.
+
+    This reponse object should be used by views that dispatch asynchronous tasks. Most common use
+    case is for sync and publish operations. When the task_group_id is specified, the task_results
+    list should only contain 10 items. Task groups can consist of many tasks, so returning their
+    ids should occur in a paginated fashion.
+    """
+
+    def __init__(self, task_results, task_group_id=None):
+        """
+        :param task_results: List of AsyncResult objects that are used to generate the response.
```

Comment:
Can this docstring get the :class: additions with the full path to the celery AsyncResult object?